src/styles/settings/_defaults.css

```
@@ -0,0 +1,7 @@
+:root {
+  --base-font-family: 'Monteserrat', sans-serif;
+  --base-font-size: 16 px;
```

É comum sempre usar a unidade de medida sem espaço do numeral:
```suggestion
  --base-font-size: 16px;
```